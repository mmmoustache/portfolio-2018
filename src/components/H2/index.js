import styled from "styled-components";
import { convertRemToPixels as rem } from '../../utils/utils';
import { Breakpoints } from '../../utils/breakpoints';

const marginBottom = `${rem('20px')}`;

export default styled.h2`
  font-size: ${rem('40px')};
  letter-spacing: -${rem('2px')};
  margin: 0 0 ${marginBottom};

  @media ${Breakpoints.medium} {
    font-size: ${rem('60px')};
    letter-spacing: -${rem('3px')};
  }
`;
