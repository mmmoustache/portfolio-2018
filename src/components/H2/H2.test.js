import React from 'react';
import renderer from 'react-test-renderer';
import H2 from "../H2";

describe('Component: H2', () => {

  it('Should render successfully', () => {
    const component = renderer.create(
      <H2>Test</H2>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

});
