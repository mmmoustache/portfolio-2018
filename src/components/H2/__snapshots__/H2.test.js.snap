// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: H2 Should render successfully 1`] = `
<h2
  className="sc-bdVaJa kujZxV"
>
  Test
</h2>
`;
